sites/funcorporate/src/backend/templates/courses/cms/course_run_detail.html

```diff
@@ -0,0 +1,4 @@
+{% extends "courses/cms/course_run_detail.html" %}
```

Comment:
Is it relevant to override `course_run_detail` template?

Reply:
ditto ;-)